src/quo/components/list/item.cljs

```diff
@@ -124,30 +124,32 @@
 
 (defn left-side [props]
   [rn/view {:style {:flex-direction :row
-                    :flex           1
+                    :flex-grow      1
+                    :flex-shrink    5
                     :align-items    :center}}
    [icon-column props]
    [title-column props]])
 
 (defn right-side [{:keys [chevron active accessory accessory-text]}]
   (when (or chevron accessory)
-    [rn/view {:style {:align-items    :center
-                      :flex-direction :row}}
-     [rn/view {:style (:tiny spacing/padding-horizontal)}
+    [rn/view {:style {:align-items     :center
+                      :flex-grow       2
+                      :flex-shrink     1
+                      :flex-direction  :row
+                      :justify-content :flex-end}}
+     [rn/view
       (case accessory
         :radio    [controls/radio {:value active}]
         :checkbox [controls/checkbox {:value active}]
         :switch   [controls/switch {:value active}]
         :text     [text/text {:color           :secondary
+                              :ellipsize-mode  :middle
                               :number-of-lines 1}
                    accessory-text]
         accessory)]
      (when (and chevron platform/ios?)
-       [rn/view {:style {:padding-right (:tiny spacing/spacing)}}
-        [icons/icon :main-icons/next {:container-style {:opacity         0.4
-                                                        :align-items     :center
-                                                        :justify-content :center}
-                                      :resize-mode     :center
+       [rn/view
```

Comment:
could you elaborate on  this change ?

Reply:
I had to remove this container style to offset the effect of adding `justify-content :flex-end` (line 139). 

The padding caused the right cheveron icon to be oddly placed because how the flex-box grew and shrank. I had to remove the additional padding so that the flex model can decide the alignment. There was some trial and error involved.

After reading this again, I feel that `:align-items` and/or `:justify-contents` to `:center` should not be truncated. I'll re-check.

Reply:
so then two more points:
1) we need this view anymore it just empty, the rule - fewer views is always better
2) so we lost padding for icon ?

Reply:
1. View is redundant, I already deleted it locally, will push soon.
2. Yes we did. But with the updated flex-box model, the `right-item` was placed with ample padding. It seemed fine to me visually, but having seen the Quo component, I realize I was wrong. I'll fix it.